### DISCUSSION TITLE
How can I request for someone at homebrew to add a cask

### DISCUSSION THREAD

**Discussion by jeddtdc** [Homebrew/discussions#1040] [Casks] [answered]
Title

**Comment by Rylan12** [comment, marked as answer]
The best way to get something added is to do it yourself. We'll be happy to help if you get stuck.

See the [Cask docs](https://github.com/Homebrew/homebrew-cask/blob/master/CONTRIBUTING.md) and, specifically, the [Adding a Cask](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md) document for help getting started.
